ui/raidboss/data/04-sb/dungeon/ghimlyt_dark.txt

```diff
@@ -0,0 +1,225 @@
+### THE GHIMLYT DARK
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~~~~~~~~~~~#
+# MAGITEK COLOSSUS #
+#~~~~~~~~~~~~~~~~~~#
+
+# -ii 394F 3775
+
+0 "Start" sync / 00:0839:The Field Of Dust will be sealed off/ window 0,1
+12.9 "Jarring Blow" sync /:Mark III-B Magitek Colossus:376E:/
+15.0 "--sync--" sync /:Mark III-B Magitek Colossus:3771:/
+22.1 "Wild Fire Beam" sync /:Mark III-B Magitek Colossus:3772:/ window 30,30
+28.8 "Magitek Slash x5" sync /:Mark III-B Magitek Colossus:3774:/ duration 8
+45.5 "Exhaust" sync /:Mark III-B Magitek Colossus:3770:/
+55.7 "Ceruleum Vent" sync /:Mark III-B Magitek Colossus:3773:/
+
+68.0 "Magitek Slash x5" sync /:Mark III-B Magitek Colossus:3774:/ duration 8
+80.2 "Magitek Slash x5" sync /:Mark III-B Magitek Colossus:3774:/ duration 8
+97.4 "Magitek Ray" sync /:Mark III-B Magitek Colossus:376F:/
+102.6 "Exhaust" sync /:Mark III-B Magitek Colossus:3770:/
+116.8 "Ceruleum Vent" sync /:Mark III-B Magitek Colossus:3773:/ window 20,20
+120.0 "--sync--" sync /:Mark III-B Magitek Colossus:3771:/
+127.1 "Wild Fire Beam" sync /:Mark III-B Magitek Colossus:3772:/
+133.1 "Magitek Ray" sync /:Mark III-B Magitek Colossus:376F:/
+138.3 "Exhaust" sync /:Mark III-B Magitek Colossus:3770:/
+143.7 "Exhaust" sync /:Mark III-B Magitek Colossus:3770:/
+156.0 "Jarring Blow" sync /:Mark III-B Magitek Colossus:376E:/ window 30,30
+162.1 "Ceruleum Vent" sync /:Mark III-B Magitek Colossus:3773:/
+169.3 "Ceruleum Vent" sync /:Mark III-B Magitek Colossus:3773:/
+
+176.5 "Magitek Slash x5" sync /:Mark III-B Magitek Colossus:3774:/ jump 68.0
+188.7 "Magitek Slash x5"
+205.9 "Magitek Ray"
+211.1 "Exhaust"
+
+#~~~~~~~~~~~~#
+# PROMETHEUS #
+#~~~~~~~~~~~~#
+
+# Unfortunately there are no log lines for when Prometheus fires after using Heat.
+
+# -ic "Lyse" "Pipin Of The Steel Heart" "Schola Centurion" "Schola Laquearius" "Schola Signifer"
+# -ii 3AC6
+
+1000.0 "Start" sync /00:0839:The Impact Crater will be sealed off/ window 1000,1
+1012.2 "Nitrospin" sync /:Prometheus:3455:/
+1023.4 "Needle Gun/Oil Shower" sync /:Prometheus:(345A|3456):/
+1031.1 "--untargetable--"
+1031.2 "Tunnel" sync /:Prometheus:3457:/ window 31.2,5
+1042.8 "Heat" duration 4 # sync /:Prometheus:3458:/
+1048.0 "--targetable--"
+
+1056.2 "Unbreakable Cermet Drill" sync /:Prometheus:3459:/ window 30,30
+1068.3 "Needle Gun/Oil Shower" sync /:Prometheus:(345A|3456):/
```

Comment:
Is the first one always needle gun? That's what I read your comments as.

Reply:
I've always seen Needle Gun first, but based on the times I saw Needle Gun second or not, I think it's at least *possible* to see a double Oil Shower. (It may not be worth accounting for, since in theory the tank should always be in front.)